Wordle.UI/Program.cs

```diff
@@ -14,12 +14,17 @@ public static void Main(string[] args)
             string secretWord = "ADEPT";
             WordScore[] wordScoreArray = new WordScore[6];
             var game = new Game();
+            var guessValidator = new GuessValidator();
             DisplayEmptyBoard();
 
             Console.WriteLine("Type in your 5 letter guess, then hit enter:");
             var guess = Console.ReadLine();
 
-            if (guess != null)
+            if (guess == null || !guessValidator.IsValid(guess))
+            {
+                throw new ArgumentException(String.Format("{0} is not a valid word", guess),"guess");
+            }

```

Comment:
The git diff did not show the entire block, I see the catch now.  I think it can be refactored, but let's not worry about that here, unless you want to.